index.ts

```diff
@@ -75,15 +75,8 @@ import readChangesets from "@changesets/read";
     console.log(
       "No changesets found, attempting to publish any unpublished packages to npm"
     );
-    let workspaces = await getWorkspaces({
-      tools: ["yarn", "bolt", "pnpm", "root"]
-    });
-
-    if (!workspaces) {
-      return core.setFailed("Could not find workspaces");
```

Comment:
`getPackages` always returns something, even if its an empty array (or throws, of course, but that's just handled by the top-level catch here)